iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/provisioning/ProvisioningTests.java

```diff
@@ -119,6 +119,21 @@ public void individualEnrollmentGetAttestationMechanismTPM() throws Provisioning
         assertNotNull(retrievedTpmAttestation.getEndorsementKey());
     }
 
+    @Test
+    public void recreateProviderWithKeyTPM() throws Exception
+    {
+        assumeTrue("Skipping because this test is only applicable to TPM attestation", testInstance.attestationType == AttestationType.TPM);
+
+        //The test protocol has no bearing on this test since it only uses the provisioning service client, so the test should only run once.
+        assumeTrue(testInstance.protocol == MQTT);
+
+        basicRegistrationFlow(EnrollmentType.INDIVIDUAL);
+        String authenticationKey = testInstance.provisioningStatus.provisioningDeviceClientRegistrationInfoClient.getTpmRegistrationResult().getAuthenticationKey();
+
+        SecurityProviderTPMEmulator securityProvider = SecurityProviderTPMEmulator.createProviderFromKey(authenticationKey);
+        DeviceClient deviceClient = new DeviceClient(testInstance.provisionedIotHubUri, testInstance.provisionedDeviceId, securityProvider, IotHubClientProtocol.MQTT);
```

Comment:
No open call after instantiating the client? How would you know that the security provider works here?